### PR TITLE
applications: asset_tracker_v2: add help to PVT/NMEA choice

### PR DESCRIPTION
**Pull request by coderbyheart**
This adds a help text to clarify what the options mean.

### CODE REVIEW

File: applications/asset_tracker_v2/src/modules/Kconfig.gps_module [hunk 1]
```diff
@@ -22,9 +22,15 @@ choice
 
 config GPS_MODULE_PVT
 	bool "Include PVT data in GPS fix event"
+	help
+	  PVT stands for position, velocity, and time.
+	  The GPS data will be send as a struct in the update.
```
Comment (simensrostad) [inline applications/asset_tracker_v2/src/modules/Kconfig.gps_module:27]:
```suggestion
	  The GPS data will be included as a struct in the GPS_EVT_DATA_READY event propagated by this module upon a fix.
```

File: applications/asset_tracker_v2/src/modules/Kconfig.gps_module [hunk 2]
```diff
@@ -22,9 +22,15 @@ choice
...
 
 config GPS_MODULE_NMEA
 	bool "Include NMEA data in GPS fix event"
+	help
+	  NMEA is a defined ASCII representation of GPS data.
+	  The GPS data will be send as a string in the update.
```
Comment (simensrostad) [inline applications/asset_tracker_v2/src/modules/Kconfig.gps_module:33]:
```suggestion
	  The GPS data will be included as a string in the GPS_EVT_DATA_READY event.
```

**Review by simensrostad — CHANGES_REQUESTED** [review]

**Review by simensrostad — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Comment by coderbyheart** [comment]
Rebased.

**Comment by coderbyheart** [comment]
Could someone merge this?

**Pull request merged**